AmericanMedicalAssociation/ama-style-guide-2
# EWL-10318: js cookie update

## JIRA Ticket(s)
- [EWL-10318: js cookie update](https://issues.ama-assn.org/browse/EWL-10318)


## Description:
updated $. cookie to Cookies.get/set and added cookie name


## To Test:
- pull the d8 PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/4194
- pull and serve. Verify that there are no undefined cookie errors in the console.
- verify that if the alert banner is closed, it does not display again


## Visual Regressions
N/A


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
